django_project/lesson/models/answer.py

```diff
@@ -0,0 +1,45 @@
+# coding=utf-8
+"""Answer model definitions for lesson apps.
+
+"""
+import logging
+from django.db import models
+from django.utils.translation import ugettext_lazy as _
+
+from lesson.models.worksheet_question import WorksheetQuestion, Worksheet
+logger = logging.getLogger(__name__)
+
+
+class Answer(models.Model):
+    """Answer lesson model."""
```

Comment:
Each lesson can have zero or more questions associated with it. The question will have one or more answers and the answer will have an indcation if it is a correct answer or not. One question could have more than one correct answers (or no correct answers).